src/Illuminate/Broadcasting/Broadcasters/Broadcaster.php

```diff
@@ -233,12 +233,13 @@ protected function isImplicitlyBindable($key, $parameter)
      * Format the channel array into an array of strings.
      *
      * @param  array  $channels
+     * @param  string  $prefix
      * @return array
      */
-    protected function formatChannels(array $channels)
+    protected function formatChannels(array $channels, $prefix = '')
```

Comment:
We can't break the method signature here on a minor release. Can this be solved any other way? Otherwise this will have to go to the master branch for the next release.

Reply:
Even if the parameter is optional?
I could also override `formatChannels` in `RedisBroadcaster` with a call to parent and then apply the prefix. What do you think about that?

Reply:
@driesvints should be resolved with my last commit.

Reply:
> Even if the parameter is optional?
Yeah, if a downstream implementation of the broadcaster implements the method it'll break execution.

Reply:
Right, makes sense :man_facepalming: